### ISSUE TITLE
obj1.adt marked as wdt

### ISSUE THREAD

**Issue by Luzifix** [Marlamin/wow.tools#59]
Hi,

obj1.adt marked as wdt file type:
https://wow.tools/files/#search=type%3Awdl&page=1&sort=0&desc=desc

**Comment by Marlamin** [comment]
Yeah, the chunks on top of that file are sadly really similar to eachother so type guessing confuses them. It's on the list to fix (after the files receive a name) but will leave it open here as well.
